cfme/common/vm_views.py

```diff
@@ -78,10 +81,13 @@ class JSInstanceEntity(JSBaseEntity):
     def data(self):
         data_dict = super(JSInstanceEntity, self).data
         if 'quadicon' in data_dict and data_dict['quadicon']:
```

Comment:
**REQUIRED**
Just add one more condition and remove try-except block.
```python
if data_dict and 'quadicon' in data_dict and data_dict['quadicon']:
```

Reply:
as far as i can tell it should be condensed down to `data_dict.get('quadicon')` - the `IndexError looks like it is related to the ux access - and cant be fixed with the extra ckeck

Reply:
@RonnyPfannschmidt I have removed those changes, latest commit doesn't contain the extra check instead I have included a try and except block to check IndexError and return empty dict.